rust/clients/orchestrator/src/admin_client.rs

```diff
@@ -14,9 +13,8 @@ pub async fn run(admin_creds_path: &Option<PathBuf>, nats_url: String) -> anyhow
 
     let credentials = admin_creds_path
         .to_owned()
-        .map(|creds| vec![Credentials::Path(creds)]);
-
-    // .ok_or(anyhow!("Failed to locate admin credential path."))?;
+        .map(|p| vec![Credentials::Path(p)])
+        .filter(|c| !c.is_empty());
```

Comment:
```suggestion
        .map(|p| vec![Credentials::Path(p)]);
```

since `Option::map` is only effective on `Some` values